core/modules/user/user.admin.inc

```diff
@@ -900,3 +620,315 @@ function user_admin_role_delete_confirm_submit($form, &$form_state) {
   $form_state['redirect'] = 'admin/config/people/roles';
 }
 
+/**
+ * Form builder; Configure user email settings for this site.
+ *
+ * @ingroup forms
+ */
+function user_settings_email($form, &$form_state) {
+  $config = config('system.core');
+  $mail_config = config('user.mail');
+  $form['email_title'] = array(
+    '#type' => 'item',
+    '#title' => t('Emails'),
+  );
+
+  $form['help'] = array(
+    '#markup' => '<p>' . t('Use this form to customize the emails sent by your website.
+      Each template may use tokens to populate special values such as a user
+      account name or the password reset link.') . '</p>',
```

Comment:
This should all be on one line. Although it makes reading code a little harder, translators have to match the string exactly, including whitespace and new lines. So translated strings are never wrapped.